bokehjs/src/lib/models/mappers/color_mapper.ts

```diff
@@ -80,4 +81,9 @@ export abstract class ColorMapper extends Mapper<Color> {
 
   protected abstract _v_compute<T>(xs: ArrayableOf<uint32 | Factor>,
     values: Arrayable<T>, palette: Arrayable<T>, colors: {nan_color: T}): void
+
+  protected _v_compute_uint32(xs: ArrayableOf<uint32 | Factor>, values: Arrayable<uint32>,
```

Comment:
The original `_v_compute` is generic and is called from a number of different places. This new `_v_compute_uint32` is a specialisation of that function for `uint32`, i.e. colors that have been converted to RGBA bytes. This is only ever called from the `rgba_mapper()` function.